variants/pca10100/variant.h

```diff
@@ -0,0 +1,155 @@
+ /*
```

Comment:
file has not been double checked, it just follows the same structure as the other dev kit variants.